worker-jobtracking/src/main/java/com/hpe/caf/worker/jobtracking/JobTrackingWorkerReporter.java

```diff
@@ -277,7 +282,7 @@ private static String getFailureDetailsString(final JobTrackingWorkerFailure rej
      * The SQL Error Code should be a 5 letter code - the first 2 characters denote the class of the error and the final 3 indicate the
      * specific condition.
      */
-    private static boolean isSqlStateIn(final SQLException ex, final String... errorClasses)
+    private static boolean isSqlStateIn(final SQLException ex, final List<String> errorCodes, final String... errorClasses)
```

Comment:
This usage of this function appears awkward now.  How about you put the signature back the way it was and determine whether it is an error class or code by checking the length?